components/class-m-chart-admin.php

```diff
@@ -421,9 +433,7 @@ public function save_post( $post_id ) {
 			&& m_chart()->is_valid_library( $_POST[ m_chart()->slug ]['library'] )
 		) {
 			// Load the library in question in case there's a filter/action we'll need
-			if ( 'highcharts' == $_POST[ m_chart()->slug ]['library'] ) {
-				m_chart()->highcharts();
-			}
+			m_chart()->library( $_POST[ m_chart()->slug ]['library'] );
```

Comment:
I'd suggest sanitizing `$_POST[ m_chart()->slug ]['library']` before passing it into the method.

Reply:
That being said, I guess it isn't as necessary due to the check in the conditional. I just get the heebie jeebies when I see `$_POST` vars being unsanitized and then being passed around.

Reply:
Hmm, yeah, I'm trying to think about how to best sanitize it. I guess I was thinking of the conditional check as being essentially the same thing.